core/src/test/scala/org/apache/spark/rpc/TestRpcEndpoint.scala

```diff
@@ -26,15 +26,15 @@ class TestRpcEndpoint extends ThreadSafeRpcEndpoint with TripleEquals {
 
   override val rpcEnv: RpcEnv = null
 
-  @volatile private var receiveMessages = ArrayBuffer[Any]()
+  @volatile private val receiveMessages = ArrayBuffer[Any]()
```

Comment:
We can delete @volatile too.

Reply:
good catch, thanks  @beliefer, but I think we can batch clean up invalid `@volatile` in a separate pull request, this might not be the only case :)

Reply:
Sounds reasonable to me.

Reply:
It seems you must fix them. :)
```
[error] /home/runner/work/spark/spark/core/src/test/scala/org/apache/spark/rpc/TestRpcEndpoint.scala:29:25: values cannot be volatile
[error]   @volatile private val receiveMessages = ArrayBuffer[Any]()
```

Reply:
Got, thank you @beliefer , for safety reasons, I choose to keep this code as it is.